### PR TITLE
Kazaz/remove operational treasury

### PR DESCRIPTION
**Pull request by kazazor**
## What's the purpose of your PR?

- [ ] Fixing a bug
- [ ] A new Feature
- [X] Techdebt
- [ ] General Task

## What is your PR all about?
* Remove any code references to the old operational treasury and old treasury
* Kept one task in todo to be deleted once I'll run the task to change the operational registry to the treasury

### CODE REVIEW

**Comment by kazazor** [comment]
@tkeidar I added the don't merge label, but you can still review everything. It's there just because I want to run the task and then delete it from the file

File: hardhat.config.ts [hunk 1]
```diff
@@ -95,9 +90,6 @@ const config: HardhatUserConfig = {
     BOT: {
       default: 2
     },
-    OPERATIONAL_TREASURY: {
-      default: 3
-    },
     TEST1: {
       default: 4
```
Comment (tkeidar) [inline hardhat.config.ts:94]:
```suggestion
      default: 3
```

Change also the counter of the accounts below

File: utils/DeployedContractsHelpers.ts [hunk 2]
```diff
@@ -130,22 +130,15 @@ export async function getDeployer(hre: HardhatRuntimeEnvironment) {
...
-
-export function getOperationalTreasury(hre: HardhatRuntimeEnvironment): Promise<SignerWithAddress> {
-  return hre.ethers.getNamedSigner("OPERATIONAL_TREASURY");
+  return await hre.ethers.getNamedSigner("BOT");
 }
 
 export async function getUser(hre: HardhatRuntimeEnvironment): Promise<SignerWithAddress> {
-  const user = await hre.ethers.getNamedSigner("TEST2");
-  return user;
+  return await hre.ethers.getNamedSigner("TEST2");
```
Comment (tkeidar) [inline utils/DeployedContractsHelpers.ts:137]:
```suggestion
  return hre.ethers.getNamedSigner("TEST2");
```

NIT
you can remove the `async` keyword from the function and this `await` 
same for the next function

**Review by tkeidar — APPROVED** [review]
I had some small comments, fix and then you can merge

**Pull request merged**